app/helpers/application_helper.rb

```diff
@@ -27,10 +27,11 @@ def generate_breadcrumbs(current_page, previous_pages)
     tag.div(class: 'govuk-breadcrumbs') { tag.ol(crumbs.join('').html_safe, class: 'govuk-breadcrumbs__list', role: 'breadcrumbs') }
   end
 
-  def govuk_header_navigation_item(active_class = false)
-    base_classname = 'govuk-header__navigation-item'
-    classname = "#{base_classname} #{active_class ? "#{base_classname}--active" : ''}"
-    tag.li(class: classname) { yield }
+  def govuk_header_navigation_item(active_class: '', &block)
```

Comment:
`active_class` it's a string (check the caller!), so the default should be a string too, empty as default and not a boolean.